docs/hw/hookup.md

```diff
@@ -3,6 +3,39 @@
 The iRobot® Create® 3 has a USB-C®[^1] connector implementing a USB 2.0 host capable of 5 V @ 3 A continuous, which can be used to power and communicate with various downstream devices.
 Here are some hookup examples.
 
+## NXP NavQ+
```

Comment:
Please put this section after the NVIDIA Jetson section, both because this is community-contributed content, and also because this section lacks proper copyrighting, and so when you add footnotes, you won't have to disrupt all the other sections.

Reply:
if there are no symbols to add, this is probably fine.

Reply:
Ok, sounds good.